### ISSUE TITLE
group-by-intervals-test fails

### ISSUE THREAD

**Issue by eerohele** [juxt/tick#22]
`make test` fails with:

```
FAIL in (group-by-intervals-test) (interval_test.cljc:555)
p and s
expected: (= {(t/year 2017) [(ti/new-interval (t/date-time "2017-12-20T00:00") (t/date-time "2018-01-01T00:00"))], (t/year 2018) [(ti/new-interval (t/date-time "2018-01-01T00:00") (t/date-time "2018-01-10T00:00"))]} (ti/group-by (t/divide (ti/bounds (t/year 2016) (t/year 2019)) t/year) [(ti/new-interval (t/date-time #inst "2017-12-20T00:00:00.000-00:00") (t/date-time #inst "2018-01-10T00:00:00.000-00:00"))]))
  actual: (not (= {#jsr310/year "2017" [#:tick{:beginning #jsr310/date-time "2017-12-20T00:00", :end #jsr310/date-time "2018-01-01T00:00"}], #jsr310/year "2018" [#:tick{:beginning #jsr310/date-time "2018-01-01T00:00", :end #jsr310/date-time "2018-01-10T00:00"}]} {#jsr310/year "2017" [#:tick{:beginning #jsr310/date-time "2017-12-20T02:00", :end #jsr310/date-time "2018-01-01T00:00"}], #jsr310/year "2018" [#:tick{:beginning #jsr310/date-time "2018-01-01T00:00", :end #jsr310/date-time "2018-01-10T02:00"}]}))
```

It looks like `(t/date-time "2017-12-20T00:00")` returns `#jsr310/date-time"2017-12-20T00:00"`, while `(t/date-time #inst "2017-12-20")` returns `#jsr310/date-time"2017-12-20T02:00"` (the same time in my local time zone).

Does `#inst` -> `LocalDateTime` conversion work correctly?

**Comment by henryw374** [comment]
Hi, 

I think #inst -> LocalDateTime conversion is working correctly - in that it's meant to be using the local zone of the vm/user. The test needs fixing.


Thanks for pointing this out!

**Comment by henryw374** [comment]
fixed by 1df2524c833dde468400c9db8cbd4ffb1dc61695
